gef-ui/api-tests/about-exporter.api-test.js

```diff
@@ -0,0 +1,26 @@
+const { MAKER } = require('../server/constants/roles');
+const { withRoleValidationApiTests } = require('./common-tests/role-validation-api-tests');
+const app = require('../server/createApp');
+const { get, post } = require('./create-api').createApi(app);
+
+const dealId = '123';
+
+describe('about exporter routes', () => {
+  describe('GET /application-details/:dealId/about-exporter', () => {
+    withRoleValidationApiTests({
+      makeRequestWithHeaders: (headers) => get(`/application-details/${dealId}/about-exporter`, {}, headers),
+      whitelistedRoles: [MAKER],
+      successCode: 200,
+      disableHappyPath: true, // TODO DTFS2-6627: remove and test happy path.
```

Comment:
I think we want a new ticket number for these TODOs, with a new ticket raised explaining the work required. @oscar-richardson-softwire did something similar for portal

Reply:
Updated